### PR TITLE
Add schema validator compose file

### PR DESCRIPTION
**Pull request by parimooa**
### What is the context of this PR?

- Introduce new docker-compose file for validator.
- update run validator script to use new docker-compose
- update `validate_test_schema.sh` script to check for AJV service status instead of Python validator service
- Fix failing schema
- [Validator PR for this change ](https://github.com/ONSdigital/eq-questionnaire-validator/pull/128)

### How to review 
Run validator and validate test schema, ensure all schemas are passing

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: scripts/run_validator.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
-docker pull onsdigital/eq-questionnaire-validator:$tag
-docker run -d -p 5001:5000 "onsdigital/eq-questionnaire-validator:$tag"
+tag=use-AJV-validator
```
Comment (parimooa) [inline scripts/run_validator.sh:2]:
```suggestion
tag=latest
```

**Review by berroar — APPROVED** [review]
Schemas are validating 👍

**Review by petechd — APPROVED** [review]
Works fine both locally and with run validator script!

File: scripts/run_validator.sh [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
+TAG=${tag} docker-compose -f docker-compose-schema-validator.yml  up -d
```
Comment (MebinAbraham) [inline scripts/run_validator.sh:3]:
```suggestion
TAG=${tag} docker-compose -f docker-compose-schema-validator.yml up -d
```

**Review by MebinAbraham — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]

**Review by petechd — APPROVED** [review]

**Pull request merged**